src/message.py

```diff
@@ -10,6 +10,9 @@
 
 help_mess = """
 ^For ^more ^information ^see ^[memes.market](https://memes.market/#info1)
+_______________________
+
+WARNING! The system will be rebooted now. The whole market is bloated and biased. The formula will be changed. All accounts will be kept but their balances will be reset to 1000 MemeCoins. New formula can guarantee natural and fair investments, without billions and trillions in a couple of days. Investors with more than 1 billion will receive some cosmetic rewards to compensate the loss. (| || || |_) The reboot will happen soon. For more information please see the website.
```

Comment:
This seems like quite a long message to include in every response by the bot. I think if you plan to make a sticky post announcing the upcoming reset, then this message in the response text could be removed, or just replaced with a very brief pointer to the post.

Reply:
Yeah. For now, I think we should just warn in our comments. I will make a sticky later!